data.yml

```
@@ -578,6 +578,13 @@ landscape:
             logo: runpod.svg
             name: RunPod
             twitter: https://twitter.com/runpod_io
+          - item:
+            crunchbase: https://www.crunchbase.com/organization/daocloud
+            description: Baize(Cloud Native AI module in DCE 5.0) is an AI computing platform. It provides a software and hardware integrated AI computing experience, integrating heterogeneous computing power, optimizing GPU performance, achieving unified scheduling and operation of computing resources, maximizing computing efficiency, reducing computing costs, and providing optimized AI development frameworks to simplify AI development and deployment, accelerating the implementation of AI applications in various industries.
```

```suggestion
            description: Baize is a Cloud Native AI module in DCE 5.0 that offers an integrated AI computing platform. It optimizes GPU performance, schedules computing resources efficiently, and provides simplified AI development frameworks to accelerate AI application deployment across industries.
```
文档站 Baize 的英文还没来得及优化，这句可以先 improve 😄